<?xml version="1.0"?>
<launch>

  <node pkg="swarm_ros_bridge" type="bridge_node" name="swarm_bridge_node" output="screen" >
    <!-- <rosparam command="load" file="$(find swarm_ros_bridge)/config/ros_topics_pc2_dense.yaml" /> -->
    <rosparam command="load" file="$(find swarm_ros_bridge)/config/ros_topics_pc2.yaml" />
  </node>
  
  <node pkg="swarm_ros_bridge" type="start_pc" name="start_pc" output="screen" />

</launch>
